src/Grid.js

```diff
@@ -165,7 +169,7 @@ Grid.prototype.redraw = function(newOptions, callback) {
   var items;
 
   window.requestAnimationFrame(function() {
-    if (this.columns !== newOptions.columns) {
+    if (this.columns !== newOptions.columns || this.columnClasses !== newOptions.columnClasses) {
```

Comment:
A redraw is a fairly expensive operation and it's best if done only when the number of columns actually change. I think this could lead to unnecessary redraws – especially, as you pointed out above, it can be solved easily with changes in CSS only.